src/components/Steps/Step/Step.module.scss

```diff
@@ -0,0 +1,114 @@
+.wrapper {
+  position: relative;
+  overflow: visible;
+}
+
+.container[role="button"] {
+  cursor: pointer;
+  transition: opacity 0.3s;
+}
+
+.tail {
+  position: absolute;
+  left: 12px;
+  top: 0;
+  height: 100%;
+  width: 1px;
+  padding: 28px 0 4px 0;
+
+  &::after {
+    height: 100%;
+    width: 1px;
+    content: "";
+    display: inline-block;
+    border-radius: 1px;
+    transition: background 0.3s;
+    background-color: var(--porcelain);
+  }
+}
+
+.contentWrapper {
+  display: flex;
+}
+
+.content {
+  max-width: 340px;
+  overflow: hidden;
+}
+
+.title {
+  line-height: 24px;
+  font-size: 16px;
+  position: relative;
+}
+
+.description {
+  font-size: 14px;
+  line-height: 20px;
+  padding-bottom: 16px;
+}
+
+.disabled {
+  &[role="button"] {
+    cursor: initial;
+  }
+
+  .title {
+    color: var(--waterloo);
+  }
+
+  .description {
+    color: var(--casper);
+  }
+}
+
+.process {
+  .title {
+    color: var(--rhino);
+  }
+
+  .description {
+    color: var(--waterloo);
+  }
+}
+
+.finish {
+  .title {
+    color: var(--rhino);
+  }
+
+  .description {
+    color: var(--rhino);
+  }
+}
```

Comment:
This is ideal case for `css variables`, since `disabled` and `status`es don't overlap. If the `.title` default color is `--rhino` then there is no need for repetitive instructions. Same is true for `.description`. In the end it can be simplified to something like this:
```scss
.title {
  // ...
  color: var(--title-color, var(--rhino));
}
.description {
  // ...
  color: var(--description-color);
}

.process {
  --description-color: var(--waterloo);
}

.finish {
  --description-color: var(--rhino);
}

.disabled {
  --title-color: var(--waterloo);
  --description-color: var(--casper);
}

// ...

```